Src/BootCamp.Chapter/Checks.cs

```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            //Lesson3.PromptInt(message);
```

Comment:
Don't leave dead code. BURN IT WITH FIRE :)!

Reply:
Done